static/app/views/performance/cache/cacheLandingPage.tsx

```diff
@@ -45,8 +55,22 @@ import {DataTitles} from 'sentry/views/starfish/views/spans/types';
 const {CACHE_MISS_RATE} = SpanFunction;
 const {CACHE_ITEM_SIZE} = SpanMetricsField;
 
+const SDK_UPDATE_ALERT = (
+  <Fragment>
+    {t(
+      `If you're noticing missing cache data, try updating to the latest SDK or ensure spans are manually instrumented with the right attributes. `
+    )}
+    <ExternalLink href={`${MODULE_DOC_LINK}#instrumentation`}>
+      {t('Read the Docs')}
+    </ExternalLink>
+  </Fragment>
+);
+
+const CACHE_ERROR_MESSAGE = 'Column cache.hit was not found in metrics indexer';
```

Comment:
I'm a little confused because we're referring to the data as `cacheHitRate`, but we're querying on `CACHE_MISS_RATE` and we display the chart as Miss Rate, is this intentional?

Reply:
Not intentional, we switching from hit rate to miss rate and I didn't update the variable names. I'll clean this up in another PR!

Wrt the line u commented on tho, the error message does make sense for cache_miss_rate, because its discover function uses `cache.hit` under the hood.
 `cache_miss_rate = count(cache.hit == 'false') / (count(cache.hit == 'false') + count(cache.hit == 'true'))

Reply:
Okay awesome. The error message line made sense to me but I had no way to highlight the cacheHit variables since they weren't part of the changes in this PR